samza-core/src/main/java/org/apache/samza/clustermanager/ContainerAllocator.java

```diff
@@ -174,13 +175,7 @@ public void run() {
    * When host-affinity is enabled, all allocated resources are buffered by the hostName as key
    *
    * If the requested host is not available, the thread checks to see if the request has expired. If it has expired
-   * then two cases are handled separately
-   *
-   * Case 1: host-affinity is enabled, looks for allocated resouces on ANY_HOST and issues a container start if available,
-   *         otherwise issues an ANY_HOST request
-   * Case 2: host-affinity is disabled, expired requests are not handled, allocator waits for cluster manager to issue
-   *         resources
-   *         TODO: SAMZA-2330 Hadle expired request for host affinity disabled case
+   * please refer to {@code containerManager#handleContainerLaunch} for constraints to handle expired requests
```

Comment:
Please update comment to provide details on what this method is doing, no forward pointers.